dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/annotations/RespectsArgumentQueryLogDays.java

```diff
@@ -35,5 +35,5 @@
     description = RespectsArgumentQueryLogDays.DESCRIPTION)
 public @interface RespectsArgumentQueryLogDays {
 
-  public static final String DESCRIPTION = "The number of days of query history to dump.";
+  String DESCRIPTION = "The number of days of query history to dump.";
```

Comment:
Shouldn't it be with `final` modifier?
```suggestion
  final String DESCRIPTION = "The number of days of query history to dump.";
```

Reply:
Or looks like it's final by default

Reply:
yes